controllers/core/pod_controller.go

```diff
@@ -165,5 +170,17 @@ func (r *PodReconciler) SetupWithManager(ctx context.Context, manager ctrl.Manag
 			PageLimit:               pageLimit,
 			ResyncPeriod:            syncPeriod,
 			MaxConcurrentReconciles: 2,
-		}).Complete(r)
+		}).Watches(&source.Kind{Type: &v1.Node{}},
```

Comment:
Right now, any node event will trigger pods to be reconciled.  We can use a predicate to limit events so they are only generated on label changes, but it may be desirable to reconcile pods on all node events.